frontend/src/features/permits/hooks/hooks.ts

```diff
@@ -383,15 +383,19 @@ export const useApplicationsInProgressQuery = ({
   take = 10,
   searchString = "",
   sorting = [],
+}: {
+  page?: number;
+  take?: number;
+  searchString?: string;
+  sorting?: SortingConfig[];
```

Comment:
But for searchString, do these have to be optional parameters?

Reply:
they don't "have" to be. They all have default values -- I guess I could see `page` and `take` maybe being required, but I would still leave the `sorting` and `searchString` as optional

Reply:
Can you make page, take and sorting required?